tests/Elewant/Webapp/Application/Controllers/ApiCommandAdoptElePHPantTest.php

```diff
@@ -26,12 +26,12 @@ public function setUp()
         $this->client = $this->adoptElePHPant($this->herdId, Breed::blackAmsterdamphpRegular());
     }
 
-    public function test_command_adopt_elephpant_returns_http_status_202()
+    public function test_command_adopt_elephpant_returns_http_status_202(): void
```

Comment:
I really don't feel like having to "properly" code all the test functions. Tests do not require the same stricness as production code in my opinion.

Reply:
I'm all for being less strict in test-classes. Let's make a list of rules that can be suppressed for these!